README.md

```diff
@@ -16,7 +16,14 @@ Download the rz-pm binary for your system on the [latest release page](https://g
 | **GithubCI**  | [![Go](https://github.com/rizinorg/rz-pm/actions/workflows/go.yml/badge.svg)](https://github.com/rizinorg/rz-pm/actions/workflows/go.yml) |
 
 # Available packages
-The official database is available [here](https://github.com/rizinorg/rz-pm-db).
+
+The official database is available [here](https://github.com/rizinorg/rz-pm-db). You can change the repo by supplying a custom `RZPM_DB_REPO_URL` environment flag like this:
+
+    $ RZPM_DB_REPO_URL=https://github.com/bunnyfoofoo/my-custom-rz-pm-db rz-pm install rz-custom-plugin
```

Comment:
Please use backticks syntax for code snippets

Reply:
Fixed and squased all fixup commits